lib/pub_sub/configuration.rb

```diff
@@ -6,7 +6,8 @@ class Configuration
                   :topics,
                   :visibility_timeout,
                   :log_level,
-                  :regions
+                  :regions,
+                  :handlers
```

Comment:
The handlers are configured here as that is the solution. But they don't need to be writeable by the client outside of the config block.